### PR TITLE
[Keyboard] Add Skelett60

### PR DESCRIPTION
**Pull request by yiancar**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/novelkeys/skelett60/skelett60.c [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "quantum.h"
+
+void led_init_ports(void) {
+    // Set our LED pins as open drain outputs
+    setPinOutputOpenDrain(LED_CAPS_LOCK_PIN);
```
Comment (fauxpark) [inline keyboards/novelkeys/skelett60/skelett60.c:20]:
```suggestion
    gpio_set_pin_output_open_drain(LED_CAPS_LOCK_PIN);
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/novelkeys/skelett60/info.json [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+        "rows": ["A2", "B9", "B8", "B5", "B4"]
+    },
+    "processor": "STM32F072",
+    "url": "https://yiancar-designs.com",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x7336",
+        "vid": "0x8968"
+    },
+    "community_layouts": ["60_ansi", "60_ansi_split_bs_rshift", "60_ansi_tsangan", "60_iso", "60_iso_split_bs_rshift", "60_iso_tsangan", "60_tsangan_hhkb"],
```
Comment (dunk2k) [inline keyboards/novelkeys/skelett60/info.json:28]:
```suggestion
    "community_layouts": ["60_ansi", "60_ansi_split_bs_rshift", "60_ansi_tsangan", "60_ansi_wkl", "60_ansi_wkl_split_bs_rshift", "60_hhkb", "60_iso", "60_iso_split_bs_rshift", "60_iso_tsangan", "60_iso_tsangan_split_bs_rshift", "60_iso_wkl", "60_iso_wkl_split_bs_rshift", "60_tsangan_hhkb"],
```

File: keyboards/novelkeys/skelett60/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    KC_CAPS,  KC_A,     KC_S,     KC_D,  KC_F,  KC_G,  KC_H,    KC_J,  KC_K,  KC_L,     KC_SCLN,  KC_QUOT,           KC_ENT,
+    KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,  KC_C,  KC_V,  KC_B,    KC_N,  KC_M,  KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT, MO(1),
+    KC_LCTL,  KC_LGUI,  KC_LALT,                       KC_SPC,                          KC_RALT,  MO(1),    KC_APP,  KC_RCTL),
+
+[1] = LAYOUT_all( /* FN */
+    KC_GRV,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_DEL,  KC_TRNS,
+    KC_TRNS,  KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_PSCR,  KC_SCRL,  KC_PAUS,  KC_UP,    KC_TRNS,  KC_TRNS,
+    KC_TRNS,  KC_LEFT,  KC_DOWN,  KC_RGHT,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_HOME,  KC_PGUP,  KC_LEFT,  KC_RGHT,            KC_TRNS,
+    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_END,   KC_PGDN,  KC_DOWN,  KC_TRNS,  KC_TRNS,
+    KC_VOLU,  KC_VOLD,  KC_MUTE,                                KC_TRNS,                                KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS)
```
Comment (dunk2k) [inline keyboards/novelkeys/skelett60/keymaps/default/keymap.c:31]:
```suggestion
    KC_GRV,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_DEL,  _______,
    _______,  _______,  KC_UP,    _______,  _______,  _______,  _______,  _______,  KC_PSCR,  KC_SCRL,  KC_PAUS,  KC_UP,    _______,  _______,
    _______,  KC_LEFT,  KC_DOWN,  KC_RGHT,  _______,  _______,  _______,  _______,  KC_HOME,  KC_PGUP,  KC_LEFT,  KC_RGHT,            _______,
    _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  KC_END,   KC_PGDN,  KC_DOWN,  _______,  _______,
    KC_VOLU,  KC_VOLD,  KC_MUTE,                                _______,                                _______,  _______,  _______,  _______)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
- add compatible Community Layouts support
- for `keymap.c`, replace `KC_TRNS` with `_______` for human readability

**Review by waffle87 — APPROVED** [review]

File: keyboards/novelkeys/skelett60/readme.md
Comment (dunk2k) [inline keyboards/novelkeys/skelett60/readme.md:1]: An image of this PCB would be helpful 🙂

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/novelkeys/skelett60/readme.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Skelett 60
+
+This is a standard 60% layout PCB. It supports VIA.
```
[the remark this answers is not in the document]
Comment (yiancar) [reply keyboards/novelkeys/skelett60/readme.md:3]: No default keymap does. Its the "tradition"

File: keyboards/novelkeys/skelett60/readme.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+- Plug In
+- Unplug
+- Release Escape
+
+### Flash
+
+- Unplug
+- Hold Escape
+- Plug In
+- Flash using QMK Toolbox or CLI (`make novelkeys/skelett60:<keymap>:flash`)
```
[the remark this answers is not in the document]
Comment (yiancar) [reply keyboards/novelkeys/skelett60/readme.md:36]: There is a Physical button but I do not recommend it as it doesnt clear the eeprom. I think my description is more straight forward. I can change to the suggestion if the majority thinks so.

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**